test/addons-napi/3_callbacks/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const addon = require(`./build/${common.buildType}/binding`);
+
+addon.RunCallback(function(msg) {
+  assert.strictEqual(msg, 'hello world');
+});
+
+const global = function() { return this; }.apply();
+
+function testRecv(desiredRecv) {
+  addon.RunCallbackWithRecv(function() {
+    assert.strictEqual(this,
+                       (desiredRecv === undefined ||
+                        desiredRecv === null) ?
+                       global : desiredRecv);
```

Comment:
Fixed in https://github.com/nodejs/node/pull/11975/commits/9c0b1517b894d0a655377d23f9e4b9943d93aa38

I added consistent error checking for this test project using a `NAPI_CALL` macro. We have plans to move that macro into a shared header so it can be used by all the `addons-napi` tests, but that isn't done yet.